File: en/en.py
nums = list(map(int,input().split(" ")))
list = []
for j in range(0,5):
    sum = 0
    for i in range(0,4):
        sum += nums[i]

    list.append(sum)

    temp = nums[0]
    del nums[0]
    nums.append(temp)

print("{} {}".format(min(list), max(list)))

File: en/En.java
import java.util.Scanner;

public class En {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int sum = 0;
        int minValue = Integer.MAX_VALUE;
        int maxValue = Integer.MIN_VALUE;

        for (int i=0; i<5; i++) {
            int number = sc.nextInt();
            sum += number;
            minValue = Math.min(minValue, number);
            maxValue = Math.max(maxValue, number);
        }

        sc.close();

        System.out.println((sum - maxValue) + " " + (sum - minValue));
    }
}